apply-pr.js

```diff
@@ -41,13 +41,26 @@ if (!process.argv[2]) {
   exitWithMsg('Usage example: git-apply-pr joyent/node#1337');
 }
 
-var args = /(.*)\/(.*)#(\d*)/i.exec(process.argv[2]);
-if (!args) {
+var enablePlusOne = false;
```

Comment:
just took a look at other parts of the code where this pattern is also used, so i changed it to fit better into the other code!

thanks for the heads up! :)